facebook/delphiFacebook/R/responses.R

```diff
@@ -190,6 +198,7 @@ load_response_one <- function(input_filename, params) {
 #' @importFrom rlang .data
 #' @export
 filter_responses <- function(input_data, params) {
+  msg_plain(paste0("Filtering data..."))
```

Comment:
all of these `msg_plain` calls with just one string don't technically need `paste0`, since you're not concatenating... doesn't matter much though

Reply:
whoops, that can be attributed to some over-zealous copy-pasting